src/gas/mod.rs

```diff
@@ -396,6 +397,12 @@ fn insert_metering_calls(
 	Ok(())
 }
 
+/// If process name section while injecting gas counter
+pub enum ProcessNames {
+	No,
+	UnsafeYes,
+}
+
```

Comment:
Would be nice to have some more comprehensive documentation for the enum variants. This was the whole point for having it.